metrics_sdk/lib/opentelemetry/sdk/metrics/aggregation/sum.rb

```diff
@@ -11,41 +11,50 @@ module Aggregation
         # Contains the implementation of the Sum aggregation
         # https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#sum-aggregation
         class Sum
-          def initialize(aggregation_temporality: :delta)
+          attr_reader :aggregation_temporality, :monotonic
```

Comment:
Added `attr_reader :monotonic`. It is not being used in the class. I _think_ it will be used later when we collect a MetricStream, but I can't quite see the big picture yet